libsrc/leddevice/schemas/schema-hd108.json

```diff
@@ -0,0 +1,41 @@
+{
+	"type":"object",
+	"required":true,
+	"properties":{
+		"output": {
+			"type": "string",
+			"title":"edt_dev_spec_spipath_title",
+			"propertyOrder" : 1
+		},
+		"rate": {
+			"type": "integer",
+			"title":"edt_dev_spec_baudrate_title",
+			"default": 1000000,
```

Comment:
Just to check, is that the standard default Baudrate that gives immediate success?